server/index.js

```diff
@@ -27,7 +27,8 @@ if (DEV) {
   const compiler = webpack([clientConfig, serverConfig])
   const clientCompiler = compiler.compilers[0]
 
-  app.use(webpackDevMiddleware(compiler, { publicPath }))
+  const options = { publicPath, stats: { colors: true } }
```

Comment:
one last thing, put the const with the other 2 conts

the idea in terms of readability is just that you see 3 app.uses in a row, and they all look the same.

putting the consts together also has the same effect.

it's small, but it's just how i do things.